hvantk/utils/file_utils.py

```
@@ -0,0 +1,63 @@
+    :param out_dir: Local directory to save the file
+    :param file_name: Name of the file to save
+
+    :return: None
+    """
+    # Validate file_name doesn't contain path traversal
+    if path.isabs(file_name) or ".." in file_name:
+        raise ValueError(f"Invalid file_name: {file_name}. Must be a simple filename without path components.")
+    
+    os.makedirs(out_dir, exist_ok=True)
+    local_path = os.path.join(out_dir, file_name)
+    logger.info(f"Downloading {url} to {local_path}")
```

Ruff (0.8.2)

Undefined name `logger`

(F821)